Css/style.css

```diff
@@ -500,30 +546,41 @@ footer.Partner h2::after {
   flex-wrap: wrap;
   width: 80%;
   justify-content: center;
+  margin: auto;
 }
 
 .partImage {
-  width: 4.5rem;
-  height: 4.5rem;
+  width: 50%;
+  height: auto;
+  margin: auto;
 }
 
 .finalLogo {
   background-color: white;
   background-size: cover;
   width: 100%;
   display: flex;
+  margin: auto;
+  align-items: center;
+  justify-content: center;
 }
 
 .finalLogoImage {
-  width: 8rem;
-  height: 8rem;
+  width: 9rem;
+  height: 4rem;
+}
+
+.featuredSpeakers .speakers li .texContainer span:hover,
+.featuredSpeakers .speakers li .texContainer span:active {
+  text-decoration: underline;
 }
 
 .finalLogo p {
-  margin-top: 3rem;
+  margin-top: 1rem;
```

Comment:
- [ ] To match the original design better, you should make this horizontally centered.

Behance design:
![image](https://user-images.githubusercontent.com/66967274/142619745-08600f78-a44e-4f63-b044-a21add328224.png)

Your design:
![image](https://user-images.githubusercontent.com/66967274/142619854-83aa8ce7-1f65-4319-bb8b-d571781312db.png)